docs/guides/data_sources.ipynb

```diff
@@ -261,6 +261,21 @@
     "S2403_C01_027E: Public administration   "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Poverty (Time Series)\n",
+    "\n",
+    "- **Definition:** This dataset contains estimations of poverty and income in the United States by county.\n",
+    "\n",
+    "- **Time Restrictions:** 1998 - 2021\n",
+    "\n",
+    "- **Source:** Obtained from the United States Census Bureau Website: [Small Area Income and Poverty Estimates (SAIPE) Program](https://www.census.gov/programs-surveys/saipe/data/datasets.html).\n",
+    "\n",
+    "- **Notes:** The dataset contains the following variables: total poverty amount and percentage value, poverty of people aged 0-17 and percentage value, median household income."
```

Comment:
Please clarify -- is povertyAllpr the percentage (maybe povertyAllprct is a more obvious name)? What are the units on each of the variables (replace "poverty amount" with "number of people in poverty", or whatever the census bureau calls it)?